react-app/src/components/ProposalDetailScreen/ProposalDetailScreenAPI.ts

```diff
@@ -26,6 +32,52 @@ const calculateTurnout = (tallyResult: Proposal["tallyResult"]) => {
   return turnout.toNumber();
 };
 
+export function useSetProposalReactionMutation(): (
```

Comment:
can we have a generic set reaction api that accepts node id as input? & reuse it across whole app?